src/communication/azext_communication/__init__.py

```diff
@@ -66,7 +66,7 @@ def _register_data_plane_account_arguments(self, command_name, arguments):
         if not command:
             return
 
-        group_name = 'communication'
+        group_name = 'Communication'
```

Comment:
Why you change the group name?

Reply:
From the feedback I received, when people are looking at the commands' helps, it looks like it should start with a capital C.